Documentation/openshift-issues.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Common Issues on OpenShift
+weight: 10620
+indent: true
+---
+
+# Common Issues on OpenShift
+
+## Monitoring Ceph Cluster on OpenShift
+
+OpenShift Console uses OpenShift Prometheus for monitoring and populating data in Storage Dashboard. Following troubleshooting guide assumes that it is allowed to monitor the Ceph Cluster.
+
+1. Check if Ceph Cluster monitoring is enabled
+
+    `$ kubectl get cephcluster rook-ceph -n rook-ceph -oyaml`
```

Comment:
```suggestion
    `$ oc get cephcluster rook-ceph -n rook-ceph -o yaml`
```

Reply:
Also why not starting with `oc project rook-ceph` so that we can avoid passing the namespace all the time.

Reply:
I thought it'd be better to stick to `kubectl`. Will all users of OpenShift use `OC` commands? Or is it optional?

Reply:
I think it's optional but to me `oc` goes along with OpenShift. Perhaps a little sentence to explain that both are available but we decided to go with `oc` in the example?

Reply:
For OpenShift topics let's use `oc` commands. OpenShift users don't use `kubectl`.

Reply:
ack.